build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.dokka.gradle.DokkaMultiModuleTask
 
 plugins {
-    id("at.asitplus.gradle.conventions") version "1.9.10+20230922" //version can be omitted for composite builds
+    id("at.asitplus.gradle.conventions") version "1.9.10+20231030" //version can be omitted for composite builds
```

Comment:
The commits contained something about Kotlin 1.9.20 ... but here it is still 1.9.10?